### ISSUE TITLE
When Gradle is executed with --continue flag, report file isn't listed at the end of the console

### ISSUE THREAD

**Issue by JLLeitschuh** [JLLeitschuh/ktlint-gradle#259]
I'm executing the `gradle/gradle` build `sanityCheck` task with the `--continue` flag.

Although the errors are reported during execution, they are not reported in the final error report printed at the end.

If you have very long logs, this means you have to dig for the resulting error.

```
1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':kotlinDsl:ktlintMainSourceSetCheck'.
> Process 'command '/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':kotlinDsl:ktlintTestSourceSetCheck'.
> Process 'command '/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

```

**Comment by Tapchicoma** [comment]
@JLLeitschuh is it really possible to print all errors in the end? :thinking: 

Basically Gradle runs many tasks with the same name and there is not "meta" task that can collect it.

**Comment by JLLeitschuh** [comment]
I'll have to think about this. I think there is a way to do this, but I don't know what. Let me talk to some co-workers and get back to this.

**Comment by ahmed3elshaer** [comment]
> I'll have to think about this. I think there is a way to do this, but I don't know what. Let me talk to some co-workers and get back to this.

any chance to get more info on this please ?

**Comment by JLLeitschuh** [comment]
I suggest we look at some other tasks that generate reports and see if they do anything interesting
